src/administration/integrations/gitlab.md

```diff
@@ -0,0 +1,72 @@
+# GitLab
+
+The [GitLab](https://gitlab.com) integration allows you to manage your
+Platform.sh environments directly from your GitLab repository.
+
+Features supported:
+
+* Create a new environment when creating a branch or opening a
+    pull request on GitLab.
+* Rebuild the environment when pushing new code to GitLab.
+* Delete the environment when merging a pull request.
+
+## Setup
+
+### 1. Generate a token
+
+To integrate your Platform.sh project with an existing GitLab
+repository, you first need to generate a token on your GitLab user
+profile. Simply go to your Settings page on GitLab and click
+`Access Tokens`.
+
+Fill the `Name` field for example with "Platform.sh Integration" and optionally
+set an expiration time.
+
+Give it a description and then ensure the token has the following scopes:
+
+ * `api`  - Access your API
+ * `read_user` - Read user information
+ * `read_registry` - Read Registry
+
+Copy the token and make a note of it (temporarily).
+
+Note that for the integration to work, your GitLab user needs to have permission to push code to the repository.
+
+### 2. Enable the integration
+
+Note that only `project owner` or `project admin` can manage the integrations.
+
+Open a terminal window (you need to have the Platform.sh CLI installed). Enable the GitLab integration as follows:
+
+```bash
+platform integration:add --type=gitlab --token=THE-TOKEN-YOU-WROTE-DOWN --base-url=https://THE-URL-OF-YOUR-GITLAB/ --gitlab-project=MY-NAMESPACE/MY-PROJECTNAME
+```
+
+Optional parameters:
+* `--fetch-branches`: Track and deploy branches (true by default)
+* `--build-merge-requests`: Track and deploy merge-requests (true by default)
+* `--merge-requests-clone-parent-data` : should merge requests clone the data from the parent environment (true by default)
+
+### 3. Add the webhook
```

Comment:
We don't add the hook automatically?

Reply:
I don't believe so. If we can that would be a very good thing (also for github etc.. there is a ticket for that)